### PR TITLE
docs: specify minimum less version

### PR DESCRIPTION
**Pull request by FoxxMD**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

ant-design#22800

### 💡 Background and solution

Recommend to use latest version of less or a minimum version to avoid any issues with compiling less files when using babel-plugin-import -- specifically useful for users who are not using CRA and may not know what version of less should be used.

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    Add less version requirements for style usage with babel-plugin-import       |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️ 

- [ ] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered docs/react/use-with-create-react-app.en-US.md](https://github.com/FoxxMD/ant-design/blob/specifyLessVersion/docs/react/use-with-create-react-app.en-US.md)

### CODE REVIEW

**Comment by FoxxMD** [comment]
I need a chinese language user to update the CN doc 😅

File: docs/react/use-with-create-react-app.en-US.md [hunk 1]
```diff
@@ -203,6 +203,8 @@ We have built-in dark theme and compact theme in antd, you can reference to [Use
 
 > You could also try [craco](https://github.com/sharegate/craco) and [craco-antd](https://github.com/FormAPI/craco-antd) to customize create-react-app webpack config same as customize-cra does.
 
+> Note: It is recommended to use the latest version of `less`, or a minimum version greater than 3.1.0
```
Comment (afc163) [inline docs/react/use-with-create-react-app.en-US.md:206]:
```suggestion
> Note: It is recommended to use the latest version of `less`, or a minimum version greater than `3.1.0`.
```

**Comment by afc163** [comment]
Thx!

**Comment by yoyo837** [comment]
中文没补。

**Comment by FoxxMD** [comment]
> 中文没补。

Yes I made a comment about that, I guess it got overlooked. @afc163 can you add the suggestion to the CN doc as well?

ALSO: It should be `3.0.1` not `3.1.0`, my mistake!

**Comment by xrkffgg** [comment]
OK, i will fix it

**Pull request merged**